### PR TITLE
TypeScript, Less, Fetching, Post

### PR DESCRIPTION
**Pull request by fstodulski**
Simple project, ready to use.
Implemented TypeScript and Less.

Also created prototype for handling fetch request.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { Component } from 'react';
+import './App.less';
+
```
Comment (fstodulski) [inline src/App.tsx:4]: Need some comment

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+interface MainProps {}
+
+interface MainState {}
+class App extends Component<MainProps, MainState> {
+  constructor(props: MainProps) {
+    super(props);
+  }
+
+  public componentDidMount() {}
```
Comment (fstodulski) [inline src/App.tsx:15]: Unused method

File: config/webpack.config.dev.js [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+      // It is guaranteed to exist because we tweak it in `env.js`
+      process.env.NODE_PATH.split(path.delimiter).filter(Boolean)
+    ),
+    // These are the reasonable defaults supported by the Node ecosystem.
+    // We also include JSX as a common component filename extension to support
+    // some tools, although we do not recommend using it, see:
+    // https://github.com/facebookincubator/create-react-app/issues/290
+    // `web` extension prefixes have been added for better support
+    // for React Native Web.
+    extensions: [
```
Comment (stanimirtt) [inline config/webpack.config.dev.js:86]: We may need only - js, jsx, ts, tsx, less for resolving.

File: config/webpack.config.dev.js [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+      '.json',
+      '.web.jsx',
+      '.jsx',
+      '.less'
+    ],
+    alias: {
+
+      // Support React Native Web
+      // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
+      'react-native': 'react-native-web',
```
Comment (stanimirtt) [inline config/webpack.config.dev.js]: Do we need that?

File: config/webpack.config.dev.js [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+    tls: 'empty',
+    child_process: 'empty',
+  },
+  // Turn off performance hints during development because we don't do any
+  // splitting or minification in interest of speed. These warnings become
+  // cumbersome.
+  performance: {
+    hints: false,
+  },
+};
```
Comment (stanimirtt) [inline config/webpack.config.dev.js]: Add new line here.

**Review by dpyskunova — APPROVED** [review]

File: tsconfig.prod.json [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "./tsconfig.json"
+}
```
Comment (stanimirtt) [inline tsconfig.prod.json]: new line

File: tslint.json [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  "extends": [],
+  "defaultSeverity": "warning",
+  "linterOptions": {
+    "exclude": [
+      "config/**/*.js",
+      "node_modules/**/*.ts",
+      "coverage/lcov-report/*.js"
+    ]
+  }
+}
```
Comment (stanimirtt) [inline tslint.json]: new line

**Review by stanimirtt — CHANGES_REQUESTED** [review]
Please look at my comments. Rename post.tsx to Post.tsx.

**Review by suezzo — APPROVED** [review]

**Review by dpyskunova — APPROVED** [review]

**Review by stanimirtt — APPROVED** [review]

**Pull request merged**